<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabImageView">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="ImageEditing">
         <attribute name="title">
          <string>Edit Character Set File</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QScrollArea" name="ScrollArea_Image_View">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
            <property name="cursor" stdset="0">
             <cursorShape>CrossCursor</cursorShape>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>1061</width>
               <height>368</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="DecfecCharacters">
         <attribute name="title">
          <string>Decfect Characterization</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QGraphicsView" name="graphicsView"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Control Panel</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>13</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="LogViewer" name="plainTextEdit">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTabWidget" name="tabThresholdProcessing">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="Advanced">
         <attribute name="title">
          <string>Advanced Threshold</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Threshold Limits</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout">
               <property name="sizeConstraint">
                <enum>QLayout::SetFixedSize</enum>
               </property>
               <item row="1" column="1" colspan="2">
                <widget class="QSpinBox" name="spinBox">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="value">
                  <number>255</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label">
                 <property name="contextMenuPolicy">
                  <enum>Qt::NoContextMenu</enum>
                 </property>
                 <property name="text">
                  <string>Upper</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" colspan="2">
                <widget class="QSpinBox" name="spinBox_2">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Lower Value</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>925</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
         <zorder>groupBox_2</zorder>
         <zorder>horizontalSpacer_2</zorder>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ImgView" name="lbMainImage_4">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>200</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Product 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Camera"/>
   <addaction name="actionSaving_Image"/>
   <addaction name="actionClose_Camera"/>
   <addaction name="actionSingle_Shot"/>
   <addaction name="actionContinuous_Shot"/>
   <addaction name="actionStop"/>
   <addaction name="actionPause"/>
   <addaction name="actionBackGround"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1313</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionOpen_Camera"/>
    <addaction name="actionClose_Camera"/>
    <addaction name="actionSingle_Shot"/>
    <addaction name="actionContinuous_Shot"/>
    <addaction name="actionStop"/>
    <addaction name="actionPause"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuCamera"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionOpen_Camera">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Icon/black-connected-24.png</normaloff>:/Icon/black-connected-24.png</iconset>
   </property>
   <property name="text">
    <string>Open Camera</string>
   </property>
   <property name="whatsThis">
    <string extracomment="Open Camera"/>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose_Camera">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Icon/black-disconnected-24.png</normaloff>:/Icon/black-disconnected-24.png</iconset>
   </property>
   <property name="text">
    <string>Close Camera</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSingle_Shot">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Icon/black-stack-of-photos-24 - Copy.png</normaloff>:/Icon/black-stack-of-photos-24 - Copy.png</iconset>
   </property>
   <property name="text">
    <string>Single Shot</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionContinuous_Shot">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Icon/black-stack-of-photos-24.png</normaloff>:/Icon/black-stack-of-photos-24.png</iconset>
   </property>
   <property name="text">
    <string>Continuous Shot</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Icon/black-stop-24.png</normaloff>:/Icon/black-stop-24.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="whatsThis">
    <string extracomment="Stop"/>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Icon/black-pause-24.png</normaloff>:/Icon/black-pause-24.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionBackGround">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Icon/background.ico</normaloff>:/Icon/background.ico</iconset>
   </property>
   <property name="text">
    <string>BackGround</string>
   </property>
  </action>
  <action name="actionSaving_Image">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Icon/save-icon-5412.png</normaloff>:/Icon/save-icon-5412.png</iconset>
   </property>
   <property name="text">
    <string>Saving Image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImgView</class>
   <extends>QLabel</extends>
   <header>imgview.h</header>
  </customwidget>
  <customwidget>
   <class>LogViewer</class>
   <extends>QPlainTextEdit</extends>
   <header>logviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
